### DISCUSSION TITLE
Missing messages / server output error / multiple result set for SQL Server

### DISCUSSION THREAD

**Discussion by Radioleao** [dbeaver/dbeaver#18889] [Q&A]
Using DBeaver on SQL Server, I have an error about missing messages and results.
Note that:
1. The error happens both with Execute SQL Statement and Execute SQL Script
2. The error happens both with or without separators.

When I run multiple queries, I get only the first result.
![image](https://user-images.githubusercontent.com/8766975/215043216-24e670ec-bbc8-40b6-ba03-6927b1516d5f.png)

When I run multiple queries and I got any error, I get only the first result and the error message is not shown.
![image](https://user-images.githubusercontent.com/8766975/215044104-6af89c8b-30c7-4d5b-8975-de93aab56909.png)

The error is not about missing execution of following statements.
I tested a temp table creation both before and after raiserror, and both tables are created.
The scripts are:
```
select 1 
select 1 as A into #t
raiserror('Test', 16, 1)

select 1 
raiserror('Test', 16, 1)
select 2 as B into #v
```
Table #t has been created as expected.
Table #v has been created as expected.

Foot Notes:
1. The issue doesn't happen when DBeaver is just started. But after some event (I have not understood which one) the issue happens everytime until I reboot DBeaver;
2. The error has already been posted ( #18816 ) but his solution didn't work for me.
3. SQL Server version is 2019

Thanks

**Comment by E1izabeth** [comment]
We will try fixing it in scope of #18890
